Code/GraphMol/FileParsers/MolFileParser.cpp

```diff
@@ -956,7 +956,7 @@ void ParseNewAtomList(RWMol *mol, const std::string &text, unsigned int line) {
     }
 
     std::string atSymb = text.substr(pos, 4);
-    atSymb.erase(atSymb.find(" "), atSymb.size());
+    atSymb.erase(atSymb.find(' '), atSymb.size());
     int atNum = PeriodicTable::getTable()->getAtomicNumber(atSymb);
     if (!i) {
       a = new QueryAtom(*(mol->getAtomWithIdx(idx)));
```

Comment:
No, @d-b-w, as far as I understand it, the sentence in which `a` gets assigned a new pointer does only execute when for `i == 0`, so that only one pointed is created. Although I agree it is a bit confusing.